src/app/core/data/mapping-collections-reponse-parsing.service.ts

```diff
@@ -0,0 +1,33 @@
+import { Injectable } from '@angular/core';
+import { ResponseParsingService } from './parsing.service';
+import { RestRequest } from './request.models';
+import { DSpaceRESTV2Response } from '../dspace-rest-v2/dspace-rest-v2-response.model';
+import { PaginatedList } from './paginated-list';
+import { PageInfo } from '../shared/page-info.model';
+import { ErrorResponse, GenericSuccessResponse, RestResponse } from '../cache/response.models';
+
+@Injectable()
+export class MappingCollectionsReponseParsingService implements ResponseParsingService {
```

Comment:
Please add TypeDocs to this new Service. This should likely also be renamed "MappedCollectionsResponseParsingRequest" (the REST endpoint is now named /mappedCollections)